### ISSUE TITLE
Create new skill

### ISSUE THREAD

**Issue by gauthamzz** [Anna-Assistant/Anna#166]
Create a new skill for Anna

What you can make is in your imagination.

You can refer to Google Assistant, Siri or Alexa Skills for Inspiration.

Look at the docs to understand [how to add a skill or feature](https://github.com/Anna-Assistant/Anna/blob/master/docs/AdditionOfNewFeatures.md) 

Basically, it's an if statement in the background.js.

Before starting to work on a new feature, please make a new issue on this repo with 
- Your Idea as Title
- How do you plan to Implement it
- What are some use cases

This is a great first issue and hard issue as well. It all depends on your imagination.

**Comment by sakshichahal53** [comment]
Hi @gauthamzz, we'd like to add a new feature- Showing Internet Speed to the user.
**Input to Anna :** "Hey, Show Internet Speed".
**Expected Ouput:** A new tab is opened directed at Fast.com showing the results.
We also plan to make Anna speak the internet speed to the user through some web-scraping techniques. 

For this, probably we need the access to the Dialogflow files to help us give a better understanding of the project, and also, to create a new skill for Anna.

**Comment by gauthamzz** [comment]
Could you make a new issue named Add Skill: Internet speed and start working on it

**Comment by sakshichahal53** [comment]
Surely. On it!

**Comment by ria-19** [comment]
Is this issue still open?

**Comment by gauthamzz** [comment]
yes

**Comment by ria-19** [comment]
I would like to work on it, please assign it to me.

**Comment by gauthamzz** [comment]
It's an open issue. If you have an idea for a skill open a new issue and I'll assign it to you
